tvision/include/tv/tvconfig.h

```diff
@@ -1,7 +1,7 @@
 /*
 
   This file isn't based in original TV file anymore.
-  It was originally based on it and modified by Robert H�hne to be used for
+  It was originally based on it and modified by Robert Hhne to be used for
```

Comment:
Please revert unintended changes

Reply:
Ugh, I've been using CodeLite as an IDE because it is available on both Linux and Windows, and the default code page is UTF-8, which I thought would handle the most widely used characters.  Turns out using Windows 437 actually does a better job. :(